### DISCUSSION TITLE
More octave controls

### DISCUSSION THREAD

**Discussion by 5-lagu** [djphazer/O_C-Phazerville#73] [Ideas]
Hey! I love experimenting with loopable random sequences, but some tone generators (in my particular case, the O.G. Winterbloom Castor & Pollux) doesn't really have any (octave-) transpose options. The C&P can do +/- one octave, but many of the CV-generators in Phazerville doesn't have any base octave controls. So if you set up i.e. Pigeons, or Shredder to generate a sequence, one can only control the root note, and not the root volt, and all the notes will be firmly in the sub-bass register. A simple root volt or octave select (at least +/- 3) would be a very welcome addition. I realize that it would mean cramming more into the UI, but maybe it could be a global setting somewhere? Maybe there already exists something like this? If so, tips are welcome too. :)

Cheers,
Lars G.

**Comment by djphazer** [comment]
This is a great idea, and I wholeheartedly agree! TB3PO has octave control built into the root note selection, and I frequently use it to achieve higher octaves on certain voices. I'll definitely try to add similar functionality to other applets... A global quantizer settings page is coming in the next update, which could potentially simplify the UI for individual applets.

**Comment by ph1-xyz** [comment]
Could you tell us more about the coming Global Quantizer? ⚡

**Reply by djphazer** [reply]
When I first picked up this project, every app or applet that dealt with pitch quantization had its own instance of the Braids quantizer. To conserve memory, I figured we only ever need FOUR global quantizer instances, and the applets can reuse them. This work has already been done in the back end.

The next step is some dedicated UI for Global Config... and then applets can select which quantizer to use: Q1, Q2, etc. I think this will make things like modal interchange easier to implement.
